src/extension/layer-filters/src/main/java/au/org/emii/geoserver/extensions/filters/LayerFiltersService.java

```diff
@@ -133,6 +133,11 @@ public void enabledFilters(HttpServletRequest request, HttpServletResponse respo
         String workspace = request.getParameter("workspace");
         String layer = request.getParameter("layer");
 
+        if(workspace==null) {
+            workspace = layer.split(":")[0];
```

Comment:
Yeah it will fall over, that's the preexisting behaviour. All the functions in the extension expect both a workspace and a layer name, so we'll have to rewrite them if we want to support layers without a workspace, which doesn't seem worth the effort.